ballerina/Package.md

```diff
@@ -0,0 +1,84 @@
+## Package overview
```

Comment:
```suggestion
## Overview
```

Reply:
In the `package.md` the usual practice is to use `Package overview`. Do we need to change that ?

Reply:
Fixed with [6b9f281010b00f8a81f1fe10af1b5e1c98b57ddf](https://github.com/ballerina-platform/module-ballerinax-stripe/pull/3/commits/6b9f281010b00f8a81f1fe10af1b5e1c98b57ddf)